src/bot.jl

```diff
@@ -1,55 +1,66 @@
-export BotConfig, snoop_bot, snoop_bench, snoopi_bot, snoopc_bot, snoopi_bench, snoopv_bench
+export BotConfig, snoop_bot, snoop_bench
 
 if VERSION <=  v"1.1"
-    isnothing(x) = x == nothing
+    isnothing(x) = x === nothing
 end
 if VERSION <=  v"1.2"
     Base.print(io, ::Nothing) = Base.print(io, "")
     Base.print(::Base.GenericIOBuffer{Array{UInt8,1}}, ::Nothing) = Base.print(io, "")
     Base.string(::Nothing) = ""
 end
+
 ################################################################
 const UStrings = Union{AbstractString,Regex,AbstractChar}
 
 """
-    BotConfig(package_name ; blacklist, os, else_os, version, else_version, package_path, precompiles_rootpath, subst, tmin)
-
-Construct a SnoopCompile bot configuration. `package_name` is the name of the package. This object is fed to the `snoopi_bot`.
+    BotConfig(package_name::AbstractString ; blacklist, os, else_os, version, else_version, package_path, precompiles_rootpath, subst, tmin)
 
-# Arguments:
-- `package_name::AbstractString`
+Construct a SnoopCompile bot configuration. `package_name` is the name of the package. This object is passed to [`snoop_bot`](@ref)
+and [`snoop_bench`](@ref).
 
 You may supply the following optional **keyword** arguments:
 
-- `blacklist` : A vector of of Strings (or RegExp) to remove some precompile sentences
+- `blacklist` : A vector of of Strings (or RegExp) to remove some precompile statements
 
-- `exhaustive`: passing `true` enables exhaustive testing of each precompilation signature, however, it slows down the processing stage. Defaults to `false`. Use this feature if SnoopCompile benchmark or your CI fails. This feature also prints the errors, which can help to find the `blacklist`s indivudally and disble `exhaustive` for the next runs.
+- `exhaustive`: passing `true` discards precompile statements that cannot be `eval`ed.
+  However, it slows down the processing stage.
```

Comment:
I'd personally take this as a sign that you're generating too many precompile statements. Consider setting `tmin` to something bigger than zero.

To be honest I don't think `exhaustive` is a very good name for this keyword, because it's not very descriptive of what it actually controls. Can we change it to `eval_succeeds` or something?

Reply:
I thought a lot about the name, I could not come up with a better name 😄

Reply:
Perhaps with a suitable `tmin` we won't need this anyway? Care to test?

Reply:
I think we should use a similar option to allow bypassing `can_eval`. See my comment here: https://github.com/aminya/SnoopCompile.jl/pull/1#discussion_r438825332